qubesdb/template-from-qubesdb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+import argparse
+import string
+
+from qubesdb import QubesDB
+
+
+def get_env(args):
+    env = {}
+    try:
+        db = QubesDB()
+        for key in args or []:
+            value = db.read(f"/vm-config/{key}")
+            if not value or len(value) == 0:
+                raise KeyError(f"Could not read from QubesDB: {key}")
+            env[key] = (value or "").decode()
```

Comment:
Is it still possible for value to be falsey here?

Reply:
Thanks for flagging this, which is a symptom of something I've raised for your input in <https://github.com/freedomofpress/securedrop-client/pull/2051#discussion_r1625008253>.